project/plugins.sbt

```diff
@@ -5,41 +5,28 @@ resolvers += "Typesafe Repository" at "https://repo.typesafe.com/typesafe/releas
 resolvers += Resolver.url("sbt-plugins", url("https://dl.bintray.com/ssidorenko/sbt-plugins/"))(Resolver.ivyStylePatterns)
 
 addSbtPlugin("org.scala-js" % "sbt-scalajs" % "0.6.26")
+addSbtPlugin("org.lyranthe.sbt" % "partial-unification" % "1.1.0")
+
+/* WartRemover does not seem to used in this project.
+We can re-enable here if we want to enable it in our build settings. */
+// addSbtPlugin("org.wartremover" % "sbt-wartremover" % "2.4.1")
```

Comment:
that's right. The code is pretty simple there so it's ok for now but we should consider enabling it there